### PR TITLE
Add FSA plan type

### PR DESCRIPTION
**Pull request by fanchen1988**
https://rippling.atlassian.net/browse/INSURANCE-68090

### CODE REVIEW

File: flux_sdk/flex/capabilities/data_models.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+# specify the plan type for FSA
+class FSAPlanType(Enum):
```
Comment (vguptarippling) [inline flux_sdk/flex/capabilities/data_models.py:5]: I am not sure if we are following this convention to add data_models inside capabilities @pshanmukrao @stevevls thoughts on this ?

Comment (fanchen1988) [reply flux_sdk/flex/capabilities/data_models.py:5]: This is for transferring capabilities data, and both enrollment and contribution will use it

**Review by Kingpin007 — APPROVED** [review]

File: flux_sdk/flex/capabilities/data_models.py (hunk 1 above)
Comment (pshanmukrao) [reply flux_sdk/flex/capabilities/data_models.py:5]: We probably want Kit level data_models in that case.

See: https://github.com/Rippling/flux-sdk/blob/main/flux_sdk/etl/data_models/schema.py

Comment (fanchen1988) [reply flux_sdk/flex/capabilities/data_models.py:5]: It's not the Kit level, only FSA could use it @pshanmukrao

Comment (pshanmukrao) [reply flux_sdk/flex/capabilities/data_models.py:5]: Both capabilities in a Kit can use it right ? Then isn't it a "Kit" level model? 

Can you help me what you mean by only FSA could use it ?

Comment (fanchen1988) [reply flux_sdk/flex/capabilities/data_models.py:5]: Only Flex will use it, it's a Flex benefit specific data model

Comment (pshanmukrao) [reply flux_sdk/flex/capabilities/data_models.py:5]: in that case add it in 

e.g., flux_sdk/flex/data_models/flex_models.py

Comment (fanchen1988) [reply flux_sdk/flex/capabilities/data_models.py]: @pshanmukrao done

**Review by pshanmukrao — APPROVED** [review]

**Pull request merged**